e2e/suite_test.go

```diff
@@ -145,5 +145,5 @@ var _ = Describe("TopoLVM", func() {
 	Context("multiple-vg", testMultipleVolumeGroups)
 	Context("lvcreate-options", testLVCreateOptions)
 	Context("cleanup", testCleanup)
-	Context("CSI sanity", testSanity)
+	// Context("CSI sanity", testSanity)
```

Comment:
- even if it's temp commit this alone would not trigger `testSanity` why to comment out whole file?

Reply:
Keeping unreached code was also throwing out errors. So had to comment it out for the time being until we find a workaround.

Reply:
- ack, maybe adding `//nolint:deadcode,unused` at the end of func will work, anyways fine for now